thermal_eol/README.md

```diff
@@ -0,0 +1,32 @@
+# TAP MSR
+
+This directory contains a Serpent 2 Model for the Transatomic Power (TAP) Molten Salt Reactor (MSR).
+The model is based of the technical report published by Oak Ridge National Laboratory (ORNL), which can be found here https://www.osti.gov/biblio/1410921
+
+Several assumptions have been made due to lack of information.
+The density of the salt was determined using $`\rho(T)=6.105-0.001272T[K]`$
+which is based on https://aip.scitation.org/doi/10.1063/1.3253134 although the temperatures of interest are outside the support of the equation.
+
+The control rod design is based on the MSRE control rod design. 
+In order to fit the MSRE CR into the pin pitch of the TAP MSR, it was assumed that the thickness to outer diameter ratio was kept constant for all cylinders.
+The control rod length was arbitrarily set to 2/3 the height of the active core. 
+The CR guide tub and cladding were assumed to be Hastelloy
+ 
+Moderator rods have been included in all latice cell locations. 
```

Comment:
@andrewryh This is **not** what you wanted me to do but I think it supplies the same information. Because of the geometry slicing it is going to be difficult to get the salt volume fraction of the current model. I would have to go back and remove partial rods at the vessel wall to get an accurate number. Do you want me to do that or just keep this estimate

Reply:
No, do not worry about that.

Reply:
I have a script that calculates current SVF for given Serpent input.
You good, merging